README.md

```diff
@@ -1,2 +1,17 @@
 # go-http-middleware
 
+[![Build Status](https://drone.corp.mongodb.com/api/badges/kanopy-platform/go-http-middleware/status.svg)](https://drone.corp.mongodb.com/kanopy-platform/go-http-middleware)
+
+The `go-http-middleware` is a collection of generic HTTP middleware functions.
+
+## Design Guidelines
```

Comment:
It might be a bit premature to add some of these guidelines considering that we really only have one basic example at the moment. I think I'd be fine deferring this or adding more specific contribution guidelines.

Reply:
Sounds good.

Reply:
I think for contributing guidelines we should link to a generic document within the #kanopy-platform github organization.  Thoughts?

Reply:
No, generally contribution guidelines are per-repo. [GitHub has some good guidance for this.](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors)

Reply:
No, generally contribution guidelines are per-repo. [GitHub has some good guidance for this.](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors)

Reply:
Fair, perhaps I am thinking more generalized code style and best practices document that would be shared by the org.